Index: target/generated-sources/xjc/org/cip4/lib/xjdf/schema/Product.java
===================================================================
--- target/generated-sources/xjc/org/cip4/lib/xjdf/schema/Product.java
+++ target/generated-sources/xjc/org/cip4/lib/xjdf/schema/Product.java
@@ -265,6 +265,16 @@
     }

     /**
+     * Gets the value of the childRefs property <b>WITHOUT</b> lazy initialization.
+     *
+     * @see Product#getChildRefs()
+     *
+     */
+    public final List<org.cip4.lib.xjdf.schema.Product> getRawChildRefs() {
+        return this.childRefs;
+    }
+
+    /**
      * Ruft den Wert der commentURL-Eigenschaft ab.
      *
      * @return